pydantic/main.py

```diff
@@ -71,6 +72,7 @@
 
     ConfigType = Type['BaseConfig']
     Model = TypeVar('Model', bound='BaseModel')
+    ModelFieldsDict = OrderedDict[str, ModelField]
```

Comment:
Had to go with the `if TYPE_CHECKING` because of [this issue](https://mypy.readthedocs.io/en/stable/common_issues.html#using-classes-that-are-generic-in-stubs-but-not-at-runtime)